### PR TITLE
fix problem with CVE-2021-3520

### PR DESCRIPTION
**Pull request by ekomarova**
This change should help fix the problem with https://nvd.nist.gov/vuln/detail/CVE-2021-3520

### CODE REVIEW

**Comment by ekomarova** [comment]
@remkade

**Comment by remkade** [comment]
@chenghlee or @katietz It looks like upstream doesn't yet have a fix released. I'm not quite sure what our policy on this is. Do we want to maintain temporary patched version or wait until upstream releases a fix for this high severity issue?

See:
lz4/lz4#972

File: recipe/patches/CVE-2021-3520.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git a/lib/lz4.c b/lib/lz4.c
```
Comment (chenghlee) [inline recipe/patches/CVE-2021-3520.patch:2]: At the top of this patch file, I would add comment and a link to the upstream issue/PR (e.g., `See: https://github.com/lz4/lz4/pull/972`) so we know where it came from and can figure out when we should drop it. 

Other than that, I think it's okay to apply this patch and rebuild the `lz4` package.

**Review by chenghlee — CHANGES_REQUESTED** [review]

**Review by chenghlee — APPROVED** [review]

**Comment by remkade** [comment]
Before we merge this in I want to make sure it goes through our usual build and test process. I'll hold this open until this is built and released.

@skupr-anaconda and @varlackc if you have time during your PBS Review process this is a good one to build.

**Comment by skupr-anaconda** [comment]
CVE-2021-3520.patch is failed when I built the recipe.
I recommend using the format for a patch like this one https://github.com/lz4/lz4/commit/8301a21773ef61656225e264f4f06ae14462bca7.patch
I'll create a new PR with some enhancements in the recipe and comments about the patch

**Comment by skupr-anaconda** [comment]
@chenghlee @remkade The new PR for review https://github.com/AnacondaRecipes/lz4-c-feedstock/pull/2

**Comment by skupr-anaconda** [comment]
Merged https://github.com/AnacondaRecipes/lz4-c-feedstock/pull/2

**Pull request closed without merging**